docs/modules/migrate/pages/rolling-restart.adoc

```diff
@@ -0,0 +1,45 @@
+= Upgrade Cluster to {enterprise-product-name}
+:description: You can use an in-place rolling restart to move each member on your Open Source cluster to an Enterprise cluster.
+
+{description}
+
+This approach ensures that you can move your cluster without interruption to the service. 
+Similar to the rolling upgrade process, a rolling restart allows you to move one member at a time.
+
+The codebase of your {open-source-product-name} cluster must match the target {enterprise-product-name} release. 
+Currently, rolling restart can only be used to update your 5.5 {open-source-product-name} cluster members.
+
+== Best Practice
+
+Before starting a rolling restart, consider the following best practices:
+
+* Backup your IMap and ReplicatedMap data structures. This is not essential, but it is recommended practice
+* If the {open-source-product-name} member is a different version to the target {enterprise-product-name} member, update the codebase to match the target {enterprise-product-name} release
++
+**[RJL: Do we need to include any additional information to cover moving from - for example - 5.3 with CP Subsystem to Enterprise 5.5?]**
```

Comment:
This was removed from the scope, currently there is no good solution for that.

Reply:
Not ideal. And I'm not sure we can manage that expectation through the docs without it - removed question; but I'd note there are likely to be support calls around this if we cannot provide the solution 😬